Makefile

```diff
@@ -29,6 +29,23 @@ include Makefile.common
 
 DOCKER_IMAGE_NAME       ?= prometheus
 
+docker-tag-latest:
```

Comment:
would be interesting to have it in the main branch too.
I'm wondering if it would be nice to put it in the `Makefile.common` and to share it to every repo.

Reply:
Why ? Main should not push to latest. This is only so the LTS does not overwrite the latest tag

Reply:
well because like that you don't need to inject this modification in the next LTS release